### PR TITLE
Add Lucidchart

### PR DESCRIPTION
**Pull request by lnawrat**
# twofactorauth contribution PR 💪

*Checklist*
- [x] What
- [x] Why
- [x] How
- [ ] Testing
- [ ] Metrics
- [ ] Anything else?

*What?*
Adding `lucidchart.com` to the database.
https://www.lucidchart.com/

*Why?*
In order to extend the service with a missing service.

*How?*
By adding json definition and png icon.
I did not find an `svg` icon for this service.
**edit:** Thanks to @Carlgo11 , used a svg image after all.

### CODE REVIEW

**Comment by lnawrat** [comment]
The tests failed on `similarweb`, even though the website seems to be within 200K. Could it be a test error with failing similarweb request? Could someone with permissions rerun them please? :-)

image
image

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
HTTP 429 means Too Many Requests.
Just waiting a while should fix it.

In the meantime,
here's an SVG for you to use instead of the PNG:
lucidchart

**Comment by lnawrat** [comment]
@Carlgo11 , thank you. Replaced the `png` icon with the `svg` one.
Btw, can I ask which tool did you use to generate/transform png to svg?

**Comment by Carlgo11** [comment]
I downloaded the image from the front page.
Screenshot_20230227_163433
For removing the text I used Inkscape but really any vector graphics editing tool can be used.

File: entries/l/lucidchart.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Lucidchart": {
+    "domain": "lucidchart.com",
+    "contact": {
+      "twitter": "LucidSoftware",
```
Comment (Carlgo11) [inline entries/l/lucidchart.com.json]:
```suggestion
      "twitter": "LucidHelp",
```
Support accounts are generally preferred as they tend to be monitored more.

File: entries/l/lucidchart.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Lucidchart": {
+    "domain": "lucidchart.com",
```
Comment (Carlgo11) [inline entries/l/lucidchart.com.json:3]:
```suggestion
    "domain": "lucidchart.com",
    "additional-domains": [
      "lucid.app",
      "lucid.co"
    ],
```
As the authentication seems to be done at these domains, adding them will make sure the listing reaches the most amount of people :wink:

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/l/lucidchart.com.json (hunk 1 above)
Comment (lnawrat) [reply entries/l/lucidchart.com.json]: thank you 👍

File: entries/l/lucidchart.com.json (hunk 2 above)
Comment (lnawrat) [reply entries/l/lucidchart.com.json:3]: good idea, thank you 👍

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**